Packages/io.chainsafe.web3-unity/Editor/ABICSharpConverter.cs

```diff
@@ -0,0 +1,496 @@
+using System.Collections.Generic;
```

Comment:
For everyone, this class is hidden because it's one chunky boi

Reply:
Had to use brain to find this message